### PR TITLE
add infra sig first meeting agenda

### PR DESCRIPTION
**Pull request by robcxyz**
@han-so1omon - Any feedback on this agenda appreciated. I think it might be worth reviewing the inclusivity section as this is something I think would be good to make as a sort of standard for the sig meetings that invite to community. Again, all feedback welcome.

### CODE REVIEW

File: special-interest-groups/infrastructure/meetings/2023-01-x.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Infrastructure SIG Meeting 2023-01-x
```
Comment (han-so1omon) [inline special-interest-groups/infrastructure/meetings/2023-01-x.md:1]: Can you please update the infrstructure/README.md with the generalized version of the meeting information? E.g. "Meetings shall be 30 min. Meetings shall be held 1 per quarter"

File: special-interest-groups/infrastructure/meetings/2023-01-x.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- All ideas / suggestions / feedback will be treated with respect 
+- Failure to meet above will result in not attending future meetings
+
+#### SIG Charter 
+
+- See [charter](../charter.md)
+- Meeting format 
+  - Frequency -> 1 per quarter / 1 month
+  - Try to keep to 30 min 
+- Tracker frontend as a separate SIG?
```
Comment (han-so1omon) [inline special-interest-groups/infrastructure/meetings/2023-01-x.md:21]: Tracker frontend should either be specific to SIG Infrastructure or should be a Working Group (temporary cross-group collab) between SIG Infrastructure and a TBD group focused on design or UX. Likely it should stay with SIG Infrastructure and some other group can lay out specs and guidelines on how to develop UI/UX

**Review by han-so1omon — CHANGES_REQUESTED** [review]

File: special-interest-groups/infrastructure/meetings/2023-01-x.md (hunk 1 above)
Comment (robcxyz) [reply special-interest-groups/infrastructure/meetings/2023-01-x.md:1]: Done.

File: special-interest-groups/infrastructure/meetings/2023-01-x.md (hunk 2 above)
Comment (robcxyz) [reply special-interest-groups/infrastructure/meetings/2023-01-x.md:21]: Yeah that is something that I wanted to raise with participants. But I agree that UI / UX should be under another group.

**Review by han-so1omon — APPROVED** [review]

**Pull request merged**